rfcs/0036-remove-header-deps-immature-rule/0036-remove-header-deps-immature-rule.md

```diff
@@ -1 +1,32 @@
-In review, see <https://github.com/nervosnetwork/rfcs/pull/240>
+---
+Number: "0036"
+Category: Consensus (Hard Fork)
+Status: Draft
+Author: Ian Yang
+Organization: Nervos Foundation
+Created: 2021-02-07
+---
+
+# Remove Header Deps Immature Rule
+
+## Abstract
+
+This document proposes removing the loading header immature rule.
```

Comment:
Suggest to use the uniform term to refer the rule such as "Header Deps Immature Rule".

-- suggestion proposed by @chanhsu001 offline